meetings/2022-05-02.md

```diff
@@ -0,0 +1,118 @@
+# W3C Solid Community Group: Authentication Panel
+
+* Date: 2022-05-02T14:00:00Z
+* Call: https://meet.jit.si/solid-authentication
+* Chat: https://gitter.im/solid/authentication-panel
+* Repository: https://github.com/solid/authentication-panel
+
+## Present
+* Aaron Coburn
+* Abel Van den Briel
+* Laurens Debackere
+* elf Pavlik
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* Aaron Coburn
+
+---
+
+## Agenda
+
+### Issues
+
+https://github.com/solid/solid-oidc/issues
+
+### Open PRs
+
+https://github.com/solid/solid-oidc/pulls
+
+### Test Suite location
+
+### Dynamic Client Registration
+
+## Minutes
+
+### Issues
+
+* AC: We talked last week about working through the backlog of the issues and have a v0.2 to submit by the end of May.
+
+### Open PRs
+
+* AC: Making progress moving through the open issues
+* eP: Small PR submitted shortly before the meeting, clarifying terms and linking them to definitions
+
+### Test Suite location
+
+* AC: In the past we had [solid-oidc-tests](https://github.com/inrupt.solid-oidc-tests) in the `solid` organization. There was reorganization of the GitHub org, including moving most of the repos to other locations. Suggestion was to move it to the Inrupt organization. There also was a suggestion to move it into `solid-contrib`. I don't want to keep moving it around. We already point from the publish spec to the old link since it was recently moved. I think we can host it in the `solid/solid-oidc` repo, which will be as stable as editors draft itself. This way we wouldn't need to keep moving it around.
+* eP: that makes sense to me, then we can collaborate on the code
+
+ACTION: AC to make PR to add the test suite to the Solid-OIDC codebase
+
+### Dynamic Client Registration
+
+* AvdB: Noticed that Inrupt PB uses dynamic client registration (DCR) as a default, even when the issuer has disabled DCR
+* AvdB: What is the recommendation for how an app performs authentication, esp, since DCR is not part of the Solid-OIDC spec?
```

Comment:
```suggestion
* AvdB: What is the recommendation for how an app performs authentication, especially since DCR is not part of the Solid-OIDC spec?
```